### ISSUE TITLE
Imported function logs as 'null'

### ISSUE THREAD

**Issue by aguynamedben** [megahertz/electron-log#77]
Thanks for working on this library. It rocks. I got sent in a 45 minute tailspin due to this nuance.

Code:

```js
import GoogleDriveDataSource from '../dataSources/google/GoogleDriveDataSource';

log.warn(`1 - wtf is wrong ${GoogleDriveDataSource}`);
log.warn(`2 - wtf is wrong`, GoogleDriveDataSource);
log.warn(`3 - wtf is wrong`, [GoogleDriveDataSource]);
console.log(`1 - wtf is wrong ${GoogleDriveDataSource}`);
console.log(`2 - wtf is wrong`, GoogleDriveDataSource);
console.log(`3 - wtf is wrong`, [GoogleDriveDataSource]);
```

Output:

<img width="1031" alt="developer_tools_-_file____users_ben_code_tesla_app_app_html" src="https://user-images.githubusercontent.com/107841/39906637-b2ab15ec-5498-11e8-94e3-9a05d1a8fd36.png">

<img width="1034" alt="developer_tools_-_file____users_ben_code_tesla_app_app_html" src="https://user-images.githubusercontent.com/107841/39906615-9b3cdc42-5498-11e8-934a-4a887124d2bf.png">

I'm guessing the root problem is the same thing that caused this issue: https://github.com/react-community/create-react-native-app/issues/168. See this comment: https://github.com/react-community/create-react-native-app/issues/168#issuecomment-294978285. Something with using `JSON.stringify()`.

**Comment by megahertz** [comment]
Thank you. It seems you are right. It'll be fixed in v3.

**Comment by megahertz** [comment]
Fixed in v3.0.0-beta2
